client/a8c-for-agencies/sections/migrations/tag-sites-modal/add-sites-table.tsx

```diff
@@ -41,8 +48,13 @@ export default function MigrationsAddSitesTable( {
 			} else {
 				setSelectedSites( selectedSites.filter( ( id ) => id !== item.id ) );
 			}
+			dispatch(
+				recordTracksEvent( 'calypso_a8c_migrations_tag_sites_modal_select_site_click', {
```

Comment:
Should we specify which site was selected here?

Reply:
No, I think this is ok 👍